doc/LocalVMSetup.md

```diff
@@ -16,9 +16,16 @@ You can download CentOS 7.8.2003 ISO file from http://www.centos.org, or from an
 ## 3. Install a fresh CentOS 7.8.2003
 Automatic install or manual install. Basic installation is OK. When you build CORTX from source in next steps, dependant packages will be resolved and installed.
 
-## 4. You may need to add EPEL repo.
+## 4. Install GUI so you can setup ssh key
```

Comment:
It's not needed (GUI for ssh).

Reply:
Okay, noted.

Reply:
I'd suggest removing this "Install GUI so you can setup ssh key" section at all.